src/Modules/StatCan.OrchardCore.Hackathon/Drivers/HackathonDriver.cs

```diff
@@ -0,0 +1,23 @@
+using OrchardCore.ContentManagement;
+using OrchardCore.ContentManagement.Display.ContentDisplay;
+using OrchardCore.ContentManagement.Display.ViewModels;
+using OrchardCore.DisplayManagement.ModelBinding;
+using OrchardCore.DisplayManagement.Views;
+
+namespace StatCan.OrchardCore.Hackathon
+{
+    public class HackathonDriver : ContentDisplayDriver
+    {
+        public override IDisplayResult Display(ContentItem model, IUpdateModel updater)
+        {
+            if (model.ContentType == "Team")
```

Comment:
Can you change the name of the shape to be a bit more generic, to make it obvious that this is the spot to add other buttons as well ?